### ISSUE TITLE
app:lintVitalRelease Causing app build to faild

### ISSUE THREAD

**Issue by tragicmj** [react-native-checkbox/react-native-checkbox#178]
On trying to build apk, I am getting the below error
> Task :app:lintVitalRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lintVitalRelease'.
> Could not resolve all artifacts for configuration ':react-native-community_checkbox:releaseRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-community_checkbox
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
               > Read timed out

-------
 "@react-native-community/checkbox": "^0.5.9",
    "react-native": "0.66.1",

**Comment by arunshejul** [comment]
`> Task :react-native-community_masked-view:mergeReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-community_masked-view:mergeReleaseResources'.
> Could not resolve all files for configuration ':react-native-community_masked-view:releaseRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-community_masked-view
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
               > Read timed out

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings
`

**Comment by gern132** [comment]
I also had this problem this morning!

**Comment by punitkashyup** [comment]
I am also facing same issue.

**Comment by tragicmj** [comment]
@nicholaslee119 can you please look into this one. I think Jcenter dependency is causing the issue and it has been shut down since early morning today.

**Comment by Abhishektaurus** [comment]
I am also facing the same issue.
Can anyone please help ?

**Comment by andre-wells** [comment]
I ran into this problem a few weeks ago which sort of resovled itself. It started again this morning.

I believe this is because [JCenter is planned to shut down](https://learn.microsoft.com/en-us/appcenter/build/android/jcentershut).

If your app (or a dependancy) is hosted in JCenter, you're going to [start running into issues](https://blog.gradle.org/jcenter-shutdown).

> UPDATE: JFrog has decided to keep JCenter as a read-only repository indefinitely. New package and versions are no longer accepted on JCenter. All Bintray services have been shutdown.

In my case, I started removing dependencies (many were added to the project that weren't even used 🙄) to identify which dependency I needed to migrate off of.  I've been lucky; only 3 deps to get rid of and none of them were used and my builds are succeeding.

**TLDR;** Read the articles on JCenter shutting down and plan to migrate to maven hosted dependancies.

**Comment by FranticOwl** [comment]
This is a temporary workaround:

[https://github.com/facebook/react-native/issues/35136#issuecomment-1297071737](Replace all jcenter in node_modules)

**Comment by nicholaslee119** [comment]
@tragicmj 
Thanks, everyone, I will check and release this ASAP
